test/tools/libtesteth/Options.h

```diff
@@ -54,6 +54,7 @@ class Options
 	bool statediff = false;///< Fill full post state in General tests
 	bool fulloutput = false;///< Replace large output to just it's length
 	bool createRandomTest = false; ///< Generate random test
+	uint64_t randomTestSeed; ///< Define a seed for random test
```

Comment:
Can you turn this into `boost::optional<uint64_t>`?  When this option is not given it shouldn't be zero, or any number.